data/societies_and_groups/semi_secret.json

```diff
@@ -0,0 +1,146 @@
+[
+    {
+        "name": "Skull and Bones",
+        "aka": []
+    },
+    {
+        "name": "Rosicrucians",
+        "aka": []
+    },
+    {
+        "name": "Sons of Liberty",
+        "aka": []
+    },
+    {
+        "name": "Thule Society",
+        "aka": []
+    },
+    {
+        "name": "Priory of Sion",
+        "aka": []
+    },
+    {
+        "name": "Ordo Templi Orientis",
+        "aka": []
+    },
+    {
+        "name": "Knights of the Golden Eagle",
+        "aka": []
+    },
+    {
+        "name": "Woodmen",
+        "aka": []
+    },
+    {
+        "name": "Ancient Arabic Order of the Nobles of the Mystic Shrine",
+        "aka": [
+            "Shriners"
+        ]
+    },
+    {
+        "name": "Improved Order of Red Men",
+        "aka": []
+    },
+    {
+        "name": "P.E.O. Sisterhood",
+        "aka": []
+    },
+    {
+        "name": "FreeMasons",
+        "aka": [
+            "Masons"
+        ]
+    },
+    {
+        "name": "Loyal Order of Moose",
+        "aka": []
+    },
+    {
+        "name": "Bilderberg Group",
+        "aka": []
+    },
+    {
+        "name": "Ku Klux Klan",
+        "aka": [
+            "KKK"
+        ]
+    },
+    {
+        "name": "Knights of Pythias",
+        "aka": []
+    },
+    {
+        "name": "Knights of Columbus",
+        "aka": []
+    },
+    {
+        "name": "Illuminati",
+        "aka": []
+    },
+    {
+        "name": "Foresters Society",
+        "aka": []
+    },
+    {
+        "name": "Hermetic Order of the Golden Dawn",
+        "aka": []
+    },
+    {
+        "name": "Mystic Order of Veiled Prophets of the Enchanted Realm",
+        "aka": []
+    },
+    {
+        "name": "United Order of the Golden Cross",
+        "aka": []
+    },
+    {
+        "name": "United Ancient Order of Druids",
+        "aka": []
+    },
+    {
+        "name": "National Grange",
+        "aka": []
+    },
+    {
+        "name": "Ancient Order of Hibernians",
+        "aka": []
+    },
+    {
+        "name": "Masons",
+        "aka": []
+    },
+    {
+        "name": "The Trilateral Commission",
+        "aka": []
+    }
```

Comment:
Missing a comma at the end of line 115.

https://travis-ci.org/dariusk/corpora/builds/107639669#L515